athena/utils/util_functions.py

```
@@ -1,12 +1,15 @@
 """General utility functions"""
 
+from base64 import b64encode
 import concurrent.futures
 from multiprocessing import get_context
 from os import cpu_count
 from pathlib import Path
 import re
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)